contribs/gmf/less/vars.less

```diff
@@ -22,7 +22,7 @@
 @above-search-index: 6;
 
 
-@nav-width: 26rem;
+@nav-width: 400px;
```

Comment:
I'm already using 90vw but only for a small device in portrait mode.
However I agree with Stéphane to continue using rem for the width for larger screens. I changed the value from 26 to 32. It was too small.